File: SDKs/zuora.com/2021-08-20/typescript/src/sdk/models/shared/invoiceentityprefix.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// InvoiceEntityPrefix
/** 
 * Container for the prefix and starting document number of invoices.
 * 
**/
export class InvoiceEntityPrefix extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=prefix" })
  prefix?: string;

  @SpeakeasyMetadata({ data: "json, name=startNumber" })
  startNumber?: number;
}

File: SDKs/zuora.com/2021-08-20/python/src/sdk/models/shared/invoiceentityprefix.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class InvoiceEntityPrefix:
    r"""InvoiceEntityPrefix
    Container for the prefix and starting document number of invoices.
    
    """
    
    prefix: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('prefix') }})
    start_number: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('startNumber') }})
    
